### PR TITLE
extend the section on cookbooks in the manual

### PR DESCRIPTION
**Pull request by jdannberg**
This addresses the remaining points in #2109. 
I only added a short paragraph on color scales, I think if we want to recommend specific color scales for specific output quantities, this has to be a longer discussion (and is probably a separate issue).

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
   \end{verbatim}
   to insert the code in the manual.tex file.
 \item Explain what the important input parameters in this setup are, what values you
-  set them to and why. Ideally, this description includes snippets from
+  set them to and why. This paragraph should give an overview over your model setup,
```
Comment (naliboff) [inline doc/manual/manual.tex]: overview over -> overview of

File: doc/manual/manual.tex [hunk 2]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
   \end{verbatim}
   to insert the code in the manual.tex file.
 \item Explain what the important input parameters in this setup are, what values you
-  set them to and why. Ideally, this description includes snippets from
+  set them to and why. This paragraph should give an overview over your model setup, 
+  including the initial conditions, boundary conditions, geometry, etc. and anything that
```
Comment (bangerth) [inline doc/manual/manual.tex]: add a comma after "etc."

File: doc/manual/manual.tex [hunk 3]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
   \end{verbatim}
   to insert the code in the manual.tex file.
 \item Explain what the important input parameters in this setup are, what values you
-  set them to and why. Ideally, this description includes snippets from
+  set them to and why. This paragraph should give an overview over your model setup, 
+  including the initial conditions, boundary conditions, geometry, etc. and anything that 
+  is special about the setup. Ideally, this description includes snippets from
   the input file. You can place these snippets in the subfolder you created in the
   \href{doc/manual/cookbooks/.}{doc/manual/cookbooks}
   directory and include them in the manual.tex file using a command like
```
Comment (bangerth) [inline doc/manual/manual.tex]: while you're there, can you markup `\texttt{manual.tex}`?

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
...
   \begin{verbatim}
-  \lstinputlisting[language=prmfile]{cookbooks/subfolder_name/input_snippet.prm}!
+  \lstinputlisting[language=prmfile]{cookbooks/subfolder_name/input_snippet.prm}
   \end{verbatim}
 \item Show the model results in form of figures and/or plots, accompanied by an explanation
   of what happens in the model. This can also include a link to an animation of the model
   you made and uploaded somewhere, for example on youtube.
```
Comment (bangerth) [inline doc/manual/manual.tex]: YouTube

File: doc/manual/manual.tex [hunk 5]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
...
+  When creating figures or animations, you should think about the color scale that you use. 
+  Some colormaps -- like the rainbow color palette that is still the default in some 
+  visualization tools -- can obscure features present in the data and introduce 
+  artifacts, because the rainbow color scale is not preceptually uniform. For more background on this topic,
```
Comment (bangerth) [inline doc/manual/manual.tex]: -> perceptually

File: doc/manual/manual.tex [hunk 6]
```diff
@@ -10527,16 +10528,31 @@ \subsubsection{Section in the manual}
...
+  \lstinputlisting[language=prmfile]{cookbooks/subfolder_name/input_snippet.prm}
   \end{verbatim}
 \item Show the model results in form of figures and/or plots, accompanied by an explanation
   of what happens in the model. This can also include a link to an animation of the model
   you made and uploaded somewhere, for example on youtube.
+  When creating figures or animations, you should think about the color scale that you use. 
+  Some colormaps -- like the rainbow color palette that is still the default in some 
+  visualization tools -- can obscure features present in the data and introduce 
+  artifacts, because the rainbow color scale is not preceptually uniform. For more background on this topic, 
+  there is a great summary on \href{https://matplotlib.org/users/colormaps.html}{matplotlib}.
```
Comment (bangerth) [inline doc/manual/manual.tex]: just use `\url` here -- it's worth showing the whole URL here, whereas "great summary on matplotlib" seems like grammatically incorrect

**Pull request merged**